src/pudl/transform/eia923.py

```diff
@@ -825,6 +831,9 @@ def boiler_fuel(eia923_dfs, eia923_transformed_dfs):
             that page (values).
     """
     bf_df = eia923_dfs["boiler_fuel"].copy()
+    bf_df.rename(columns={"eia_sector": "sector_id_eia"}).rename(
+        columns={"sector_name": "sector_name_eia"}
+    )
```

Comment:
This rename will have no effect because you're not storing the return value. Also, I think these columns don't exist given the new naming, do they?